library/Requests.php

```diff
@@ -205,20 +204,47 @@ protected static function get_transport($capabilities = array()) {
 				continue;
 			}
 
-			$result = call_user_func(array($class, 'test'), $capabilities);
+			$result = $class::test($capabilities);
 			if ($result) {
 				self::$transport[$cap_string] = $class;
-				break;
+				return $class;
 			}
 		}
-		if (self::$transport[$cap_string] === null) {
+
+		return '';
+	}
+
+	/**
+	 * Get a working transport
+	 *
+	 * @throws Requests_Exception If no valid transport is found (`notransport`)
+	 * @return Requests_Transport
+	 */
```

Comment:
Please add a `@param` tag to the docblock and describe the format of the `$capabilities` array parameter.